core-bundle/src/Resources/contao/classes/FrontendTemplate.php

```diff
@@ -153,7 +153,7 @@ protected function compile()
 		}
 
 		// Replace literal insert tags (see #670)
-		$this->strBuffer = str_replace(array('[{]', '[}]'), array('{{', '}}'), $this->strBuffer);
+		$this->strBuffer = preg_replace('/\[{](.*)\[}]/', '{{$1}}', $this->strBuffer);
```

Comment:
```suggestion
		$this->strBuffer = preg_replace_callback('/<script[^>*]>.*?<\/script[^>*]>|\[[{}]\]/is', function ($matches) {
			return $matches[0][0] === '<' ? $matches[0] : $matches[0][1] . $matches[0][1];
		}, $this->strBuffer);
```

This should exclude script tags (untested). Maybe this is a better “workaround” for now?